python-checks/src/test/resources/checks/booleanCheckNotInverted.py

```diff
@@ -76,7 +76,7 @@ def func2():
 
 
 def check_set_as_declared_type(a: Set[str], b: Set[str]) -> None:
-    c: Set[str] = {'set_element'} # c will have type ANY
+    c: Any = "something" # c will have type ANY
```

Comment:
I don't think this change is the best we could have, as the same bug will impact it again.
I'd suggest something like:
```
c = unknown_func() # c will have type ANY
```
Then it's really clear that we can't know anything about `c`. `unknown_func` doesn't even have to be defined in the scope.